tests/system/libraries/SystemTestSpy/speechSpyGlobalPlugin.py

```diff
@@ -372,14 +372,18 @@ def wait_for_specific_speech(
 	def ensure_speech_did_not_occur(
 			self,
 			speech: str,
+			maxWaitSeconds: float,
 			afterIndex: Optional[int] = None,
-			maxWaitSeconds: float = 5.0,
 			intervalBetweenSeconds: float = 0.1,
 	) -> None:
 		"""
 		@param speech: The speech to check for.
-		@param afterIndex: Check for speech after this index. The index is exclusive.
 		@param maxWaitSeconds: The amount of time to wait in seconds.
+		As this is the expected path, this will cause a successful test run to be delayed the time provided.
+		The default for the converse, `wait_for_specific_speech`, is 5s,

```

Comment:
I meant to refer to https://github.com/nvaccess/nvda/blob/eb89962fe88ebf230baff48d70dd7c640c0a297a/tests/system/libraries/SystemTestSpy/speechSpyGlobalPlugin.py#L231

Which uses:
https://github.com/nvaccess/nvda/blob/eb89962fe88ebf230baff48d70dd7c640c0a297a/tests/system/libraries/SystemTestSpy/speechSpyGlobalPlugin.py#L76

Reply:
I've updated the function to use that constant explicitly as the default